packages/ilp-logger/src/index.ts

```diff
@@ -30,7 +30,7 @@ const createLogger = function (namespace: string) {
 interface ModuleExport {
   (namespace: string): Logger
   default: ModuleExport
-  Logger: Function
+  Logger: typeof Logger
```

Comment:
isn't a class usable as a type without requiring `typeof`?

Reply:
`Logger` would be an instance, whereas `typeof Logger` refers to its constructor